api_tests/nodes/filters/test_filters.py

```diff
@@ -94,3 +94,103 @@ def test_contributor_filter(self):
         res = self.app.get('{}{}'.format(self.contributors_url, self.user_two._id), auth=self.user.auth)
         actual = [node['id'] for node in res.json['data']]
         assert_equal(expected, actual)
+
+
+class NodesListDateFilteringMixin(object):
```

Comment:
Both of the classes in this file are mixins. Do you mean that the mixins should be at the top of the file in `test_node_list.py`?